modules/spring-config/src/main/java/org/mule/runtime/config/internal/model/ComponentModel.java

```diff
@@ -83,7 +84,7 @@
   private ComponentIdentifier identifier;
   private String componentId;
   private final Map<String, String> parameters = new HashMap<>();
-  private final Map<String, ComponentParameterAst> parameterAsts = new HashMap<>();
+  private Map<String, ComponentParameterAst> parameterAsts = new HashMap<>();
```

Comment:
initialize this as `Collections.emptyMap()`

Reply:
replaced by map + list